en/SUMMARY.md

```diff
@@ -87,6 +111,10 @@
       * [Back-transition Tuning](config_vtol/vtol_back_transition_tuning.md)
       * [VTOL w/o Airspeed Sensor](config_vtol/vtol_without_airspeed_sensor.md)
       * [VTOL Weather Vane](config_vtol/vtol_weathervane.md)
+    * [Flight Modes](flight_modes_vtol/README.md)
```

Comment:
Note, doc does not yet exist. I also haven't worked out the "story". Probably see MC for MC mode and FW for FW mode.